scripts/run-tests.sh

```diff
@@ -12,7 +12,12 @@ GO_LDFLAGS="-X github.com/infinispan/infinispan-operator/launcher.Version=${VERS
 
 go clean -testcache ./test/e2e
 if [ -z "${TEST_NAME}" ]; then
-  go test -v ./test/e2e/"${TEST_BUNDLE}" -timeout 90m -ldflags "${GO_LDFLAGS}" -parallel "${PARALLEL_COUNT}"
+  go test -v ./test/e2e/"${TEST_BUNDLE}" -timeout 90m -ldflags "${GO_LDFLAGS}" -parallel "${PARALLEL_COUNT}" 2>&1 | tee /tmp/testOutput
+  if [ ! -z ${TEST_REPORT_DIR} ]; then
+    mkdir -p ${TEST_REPORT_DIR}
+    cat /tmp/testOutput | ${GO_JUNIT_REPORT} > ${TEST_REPORT_DIR}/${TEST_BUNDLE}.xml
```

Comment:
shouldn't be this outside the `if`?

Reply:
We only want to create the xml report if `TEST_REPORT_DIR` is defined. If it's not defined, then the normal testsuite output has already been displayed via `tee` on line 15.

Reply:
:+1: